src/features/profile/components/EditPersonDialog/EditPersonFields.tsx

```diff
@@ -221,6 +221,34 @@ const EditPersonFields: FC<EditPersonFieldsProps> = ({
               value={fieldValues[field.slug]?.toString() ?? ''}
             />
           );
+        } else if (
+          field.type === CUSTOM_FIELD_TYPE.ENUM_TEXT &&
+          field.enum_choices
+        ) {
+          return (
+            <Box alignItems="flex-start" display="flex" flex={1}>
+              <FormControl fullWidth>
+                <InputLabel>{field.title}</InputLabel>
+                <Select
+                  fullWidth
+                  label={field.title}
+                  onChange={(ev) => {
+                    onChange(field.slug, ev.target.value);
+                  }}
+                  value={fieldValues[field.slug]?.toString() ?? ''}
+                >
+                  <MenuItem key="" value="">
+                    <em>None</em>
```

Comment:
Can using `<em>` like this be a bad idea? As far as I know, we don't usually use "normal HTML" in the middle MUI components like this.

Is it not possible to do this?

```suggestion
                  <MenuItem key="" value="" sx={{ fontStyle: 'italic' }}>
                    <Msg id={/* message ID goes here */}/>
```

Reply:
Fixed